File: api/code-samples/python/klaytnDebug/profiling/blockProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

file = "block.profile"
seconds = 10

sdk = OpenSDK(host)
debug_response = sdk.debug.block_profile(file, seconds)

print(json.loads(debug_response.response.data))

File: api/code-samples/javascript/klaytnDebug/profilling/blockProfile.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const file = "block.profile";
    const seconds = 10;

    sdk.debug.blockProfile(file, seconds, {}, (err, data, response) => {
        console.log(data);
    });
}
)()